.pipelines/scripts/run-e2e.sh

```diff
@@ -69,6 +69,7 @@ rm -rf kubetest2-aks
 git clone https://github.com/kubernetes-sigs/cloud-provider-azure.git
 cp -r cloud-provider-azure/kubetest2-aks .
 rm -rf cloud-provider-azure
+git config --global --add safe.directory "$(pwd)" || true
```

Comment:
How about adding a comment for this change(fix)?

Reply:
otherwise it looks good to me, thanks.

Reply:
Since tests have started, let's merge it first.